modules/renter/upload_test.go

```diff
@@ -0,0 +1,33 @@
+package renter
+
+import (
+	"testing"
+)
+
+// TestRenterSiapathValidate verifies that the validateSiapath function correctly validates SiaPaths.
+func TestRenterSiapathValidate(t *testing.T) {
+	var pathtests = []struct {
+		in    string
+		valid bool
+	}{
+		{"valid/siapath", true},
+		{"../../../directory/traversal", false},
+		{"testpath", true},
+		{"valid/siapath/../with/directory/traversal", false},
```

Comment:
It is convenient if you are doing directory traversal, or have otherwise written a script to do stuff